index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="container">
```

Comment:
Pls, use semantic tags for best practices. Instead of using the `<div>` tag, you can use the `<section>` tag as a wrapper for the container. Kindly fix this for the entire document.